Apps/WebClient/src/NewClientApp/src/constants/entryType.ts

```diff
@@ -159,4 +160,17 @@ entryTypeMap.set(EntryType.DiagnosticImaging, {
     reportEventName: "Diagnostic Imaging Exams",
 });
 
+entryTypeMap.set(EntryType.BcCancerScreening, {
+    type: EntryType.BcCancerScreening,
+    commentType: CommentEntryType.BcCancerScreening,
+    name: "BC Cancer Results",
```

Comment:
After discussion with @ninosamson, we should change the name of the dataset to "BC Cancer Screening" so it will cover both results and recalls.

Reply:
This needs to stay like this until We introduce the recalls, I had a discussion with @mk08sh about this.